views/auth/local/login.ejs

```diff
@@ -1,25 +1,55 @@
-<div class="grid-container">
-  <div class="grid-row">
+<div class="grid-container login-block">
+
+  <div class="grid-row ">
     <div class="tablet:grid-col-6 tablet:grid-offset-3 desktop:grid-col-4 desktop:grid-offset-4">
-      <h1 class="text-center">Log in</h1>
-      <% if (locals.unauthorized) { %>
-      <div class="alert alert-danger">
-        Invalid email and/or password.
+      <div class='text-center'>
+        <div>
+          <h2 class='hospital-dest-title'>Hospital</h3>
```

Comment:
This is both wierd and sloppy- you've got an opening h2 tag, and a closing h3 tag here- what did you want to use?

In the Figma design, the text is styled with h1 style. Why not just use an h1 tag, which would already inherit the font styling you need, and add a line break <br/>?

In the style for hospital-dest-title, you specify the line-height as 0.5em which doesn't make sense for me as well, I'm really not sure what you were trying to do here...?

I'm going to make this change and push it up.